integration-tests/integration/createIncident/save-additional-comment.spec.js

```diff
@@ -0,0 +1,67 @@
+const TasklistPage = require('../../pages/tasklistPage')
+const SubmittedPage = require('../../pages/submittedPage')
+const { StatementStatus } = require('../../../server/config/types')
+
+context('Add comments to statement', () => {
+  const bookingId = 1001
+  beforeEach(() => {
+    cy.task('reset')
+    cy.task('stubLogin')
+    cy.task('stubOffenderDetails', bookingId)
+    cy.task('stubLocations', 'MDI')
+    cy.task('stubOffenders')
+    cy.task('stubLocation', '357591')
+    cy.task('stubUserDetailsRetrieval', 'Mr Zagato')
+    cy.task('stubUserDetailsRetrieval', 'Mrs Jones')
+    cy.task('stubUserDetailsRetrieval', 'Test User')
+  })
+
+  it('A user can select a specific statement, add to it and then return back to statements page', () => {
+    cy.login(bookingId)
+
+    const tasklistPage = TasklistPage.visit(bookingId)
+    tasklistPage.checkNoPartsComplete()
+
+    const newIncidentPage = tasklistPage.startNewForm()
+    newIncidentPage.fillForm()
+
+    const detailsPage = newIncidentPage.save()
+    detailsPage.fillForm()
+
+    const relocationAndInjuriesPage = detailsPage.save()
+    relocationAndInjuriesPage.fillForm()
+
+    const evidencePage = relocationAndInjuriesPage.save()
+    evidencePage.fillForm()
+
+    const checkAnswersPage = evidencePage.save()
+    checkAnswersPage.clickSubmit()
+
+    const submittedPage = SubmittedPage.verifyOnPage()
+
+    const submitStatementPage = submittedPage.continueToStatement()
+    submitStatementPage.lastTrainingMonth().select('March')
+    submitStatementPage.lastTrainingYear().type('2010')
+    submitStatementPage.jobStartYear().type('1999')
+    submitStatementPage.statement().type('This is my statement')
+
+    const confirmStatementPage = submitStatementPage.submit()
+    const statementSubmittedPage = confirmStatementPage.submit()
+    const incidentsPage = statementSubmittedPage.finish()
+    const { reportId } = incidentsPage.getCompleteRow(0)
+    reportId().then(id =>
+      cy.task('getStatementForUser', { reportId: id, status: StatementStatus.SUBMITTED }).then(() => {
```

Comment:
I don't think you need to call these methods - I think this is retrieving the statement data from the database and then just ignoring it.

Also don't need to drop down the raw cypress to click the view button (this assumes that there is only one page on the screen and so is prone to breakage). 

```javascript
  const { viewButton } = incidentsPage.getCompleteRow(0)
  viewButton().click()
```

Reply:
The reason I thought these methods were still required was, the test needs a form & statement to be created in order to then have access to the statement and add to it. 
In order for the test sequence to reach the statement, the form must pass all the preceding validations, hence the reason why the tests are included.
Although ... I may be incorrect in thinking this so I tried removing the preceding form completion and validations. Because there was nothing in the db, there was no reportId() to access